### ISSUE TITLE
Vetoes from the PR author

### ISSUE THREAD

**Issue by nolar** [zalando-incubator/Zincr#21]
As an author of the PR,
I want (sometimes) to put a veto on it,
So that it would not be merged by other maintainers until some clarification is achieved.

With Zappr, it is done by putting a thumb-down (👎 ) to the comments. In Zincr & GHE, the author cannot block the PR by "Requesting the changes".

As one way (but not necessary the best), a config could be added to `.zappr.yaml` to block by existence of a label would be good (e.g. "WIP" label, or "on-hold" label).

**Comment by perploug** [comment]
Ah, so the author you want to block it - was womderig why request-changes wasn't enough for this - wouldn't the Draft state for the PR solve this - or does that come with other issues?

**Comment by nolar** [comment]
Yes, it seems, it helps to achieve the goal of vetoing the PR by its author. Zincr still reacts to these PR drafts, but this is fine (Travis also does react). Thanks.
